packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/BaseModelSqlv2.ts

```diff
@@ -2766,38 +2755,51 @@ class BaseModelSqlv2 {
     return await qb;
   }
 
-  private async extractRawQueryAndExec(qb: Knex.QueryBuilder) {
+  private async extractRawQueryAndExec(
```

Comment:
Now the method does more than just query execution so we can update the method name